pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -236,12 +236,14 @@ qtModule {
     sed 's/${lib.head (lib.splitString "-" version)} /${qtCompatVersion} /' -i "$out"/lib/cmake/*/*Config.cmake
   '';
 
+  requiredSystemFeatures = [ "big-parallel" ];
```

Comment:
For my own education, what does this do?

Reply:
On Nix level `requiredSystemFeatures` restricts doing the build on machines that are tagged by the same string (more in [Nix manual](https://nixos.org/manual/nix/stable/)).  [Apparently](https://github.com/NixOS/nixpkgs/pull/86063#issuecomment-620156069) `"big-parallel"` also get some special treatment when scheduled on Hydra.nixos.org.